packages/integration-tests/src/entities/AuthorCodegen.ts

```diff
@@ -1,51 +1,51 @@
 import {
+  Flavor,
```

Comment:
Fwiw I think this part of the diff is that I snuck in `prettier-organize-imports-plugin` in the last commit, so if you rebase / `yarn install` / run prettier, these `import` diffs should go away...